content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -12,32 +12,40 @@ from the system's components must emit [traces](/docs/concepts/signals/traces/),
 [metrics](/docs/concepts/signals/metrics/), and
 [logs](/docs/concepts/signals/logs/).
 
-Without being required to modify the source code you can collect telemetry from
-an application using [automatic instrumentation](automatic/). If you previously
-used an APM agent to extract telemetry from your application, Automatic
-Instrumentation will give you a similar out of the box experience.
-
-To facilitate the instrumentation of applications even more, you can
-[manually instrument](manual/) your applications by coding against the
-OpenTelemetry APIs.
-
-For that you don't need to instrument all the dependencies used in your
-application:
-
-- some of your libraries will be observable out of the box by calling the
-  OpenTelemetry API themselves directly. Those libraries are sometimes called
-  **natively instrumented**.
-- for libraries without such an integration the OpenTelemetry projects provide
-  language specific [Instrumentation Libraries][]
-
-Note, that for most languages it is possible to use both manual and automatic
-instrumentation at the same time: Automatic Instrumentation will allow you to
-gain insights into your application quickly and manual instrumentation will
-enable you to embed granular observability into your code.
-
-The exact installation mechanism for [manual](manual/) and
-[automatic](automatic/) instrumentation varies based on the language you’re
-developing in, but there are some similarities covered in the sections below.
-
-[instrumentation libraries]:
-  /docs/specs/otel/overview/#instrumentation-libraries
+OpenTelemetry provides [APIs and SDKs for eleven languages](/docs/languages) to
+facilitate instrumenting your code. Among others it provides the following benefits:
```

Comment:
Hmmm I think we should lay it out in terms of zero-code and code-based solutions, talk about them briefly, and then talk about the other benefits. So something to the effect of:

```
OpenTelemetry has two primary ways to instrument.

1. Zero-code solutions that, when installed, instrument libraries you use.
2. Code-based solutions via official APIs and SDKs for eleven languages.

The Zero-code solutions are great for getting started, or when you can't
modify the application you need to get telemetry out of. They provide rich
telemetry from libraries you use and/or the environment your application
runs in. Another way to think of it is that they provide information about
what's happening _at the edges_ of your application.

Code-based solutions allow you to get rich telemetry from your application
itself. They let you use the OpenTelemetry API to generate telemetry from
your application, which acts as an essential complement to the telemetry
generated by zero-code solutions.

It's generally recommended that you use both solutions when you can.

## Additional OpenTelemetry Benenfits

OpenTelemetry provides more than just zero-code and code-based telemetry
solutions. The following things are also a part of OpenTelemetry:
```

Reply:
I rather prefer leading with code-based and then letting people know that there is zero-code as well, since not all languages have zero-code and for more and more use cases code-based should be what people are doing, that's why I moved that wording around that below and also tried to give a little bit more "when to use what" beyond "start with zero code then move forward to code-based"

Reply:
I think it's fair to go with code-based first for now, but I still think we should briefly explain each before diving into the bullet-pointed list of benefits. I don't think it's evident from that list what you really get by picking one or both.